docs/transparent-validation/lab-assembly-guides/ms_z690-lab-setup.md

```diff
@@ -0,0 +1,114 @@
+# MSI PRO Z690-A WIFI DDR4 Lab Setup
+
+## Introduction
+
+This document describes assembly of MSI PRO Z690-A DDR4 lab setup. At this
+point fully assembled platform prepared using
+[Presale Device Assembly](msi_z690.md) and RTE should be ready.
```

Comment:
Why are we linking to the same documentation two times?

Reply:
Removed